README.md

```diff
@@ -64,6 +65,26 @@ match WcWidth::from_char(c) {
 }
 ```
 
+## Java usage
+
+For Java 8+, file `widechar_width.java` contains the `WcWidth` class definition, which you can use as follows:
+
+```java
+int width = WcWidth.Type.of(codePoint).defaultWidth();
+```
+
+The default values are based on the recommendations in the table of the C++ above.
+If you need a different width for some types, create your own wrapper method using something like:
+
+```java
+final var type = WcWidth.Type.of(codePoint);
+switch (type) {
+    case WIDENED_IN_9: return 1;
+    case UNKNOWN: return 0;
```

Comment:
This mention of `UNKNOWN` also needs to be removed.

Reply:
Ah, sorry, good catch. Done.